README.md

```diff
@@ -51,9 +51,39 @@ If you are using a default parser, version in your application and the JSON file
 }
 ```
 
+or
+
+```json
+{
+	"ios": {
+		"minimum_version": "1.2.3",
+		"latest_version": {
+			"version": "2.4.5",
+			"notification_type": "ALWAYS",
+		}
+	},
+	"android": {
+		"minimum_version": "1.2.3",
+		"minimum_version_min_sdk": 15,
+		"latest_version": {
+			"version": "2.4.5",
+			"notification_type": "ONCE",
+			"min_sdk":18
+		}
+	},
+	"meta": {
+		"key1": "value1",
+		"key2": "value2"
+	}
+}
+```
+
 Depending on <code>notification_type</code> property, the user can be notified <code>ONCE</code> or <code>ALWAYS</code>. The library handles this for you, and if notification type is set to <code>ONCE</code>, it will notify you via <code>onNewUpdate(String version, boolean isMandatory)</code> method only once. Every other time the library will return <code>onNoUpdate</code> for that specific version. 
 Key-value pairs under <code>"meta"</code> key are optional metadata of which any amount can be sent accompanying the required fields.
 
+Support for sdk versions have been implemented as well, so the library checks whether the user can actually receive the update based on the device that he's using. <code>minumum_version_min_sdk</code> represents the minSdk value of the minimum supported version of the application. <code>min_sdk</code> represents minSdk value of the latest version of the application.
```

Comment:
_Support for sdk versions have been implemented as well, so the library checks whether the user can actually receive the update based on the device that he's using._  -> _The library supports min sdk version. If defined, it will show a new update only if user's device is supported._